src/PaymentGateways/Gateways/PayPalCommerce/payPalCommerceGateway.tsx

```diff
@@ -277,7 +287,6 @@ import {PayPalSubscriber} from './types';
 
     const SmartButtonsContainer = () => {
         const {useWatch, useFormState} = window.givewp.form.hooks;
-        const currency = useWatch({name: 'currency'});
```

Comment:
Is it okay that we are removing the currency here and the `PaymentMethodsWrapper`?

Reply:
Okay good, thank you for the clarification.